ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -266,9 +266,17 @@ export class ReposList extends React.Component<
                                                         <i className={'icon argo-icon-' + (repo.type || 'git')} />
                                                     </div>
                                                     <div className='columns small-1'>{repo.type || 'git'}</div>
-                                                    <div className='columns small-2'>{repo.name}</div>
+                                                    <div className='columns small-2'>
+                                                        <Tooltip content={repo.name}>
+                                                            <span>{repo.name}</span>
+                                                        </Tooltip>
+                                                    </div>
```

Comment:
@vgelot I don't think I understand this change... why have a tooltip that just repeats the text?

Reply:
Sorry, it was a false manip during my rebase
I just rebased